### ISSUE TITLE
tabbing in opera doesn't work

### ISSUE THREAD

**Issue by phazei** [fnagel/jquery-ui#43]
just as the title says... open demo in opera 10.63, hit tab... nothing

**Comment by fnagel** [comment]
Im able to reproduce the issue. Im not sure about Opera and tabbing, needs further investigation.

**Comment by fnagel** [comment]
After some research Ive found out Opera handles links via STRG arrows, tab does only work for form elements. 

Ive added a tabindex="0" to make sure the links will be added to the tabindex accordingly to their DOM position. 

Should be working now, please test and report.
Thanks for pointing that out!

**Comment by fnagel** [comment]
Seems to be fixed, please re-open if necessary.
